src/components/Decks/DeckOfCards.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Deck from './Deck.js';
+
+const DeckOfCards = ({ props }) => {  
+
+  return (
+    <>
+      <Deck title={'Characters'} props={props}/>
+      <Deck title={'Gifs'} props={props}/>
```

Comment:
Currently, you have two container components, 'Gif' and 'Character', but in DeckOfCards here you render two decks, one with chars and one with gifs, which is redundant. What I would suggest doing is passing title down all the way from the top container components and then rendering one Deck in DeckOfCards that takes a prop of title and the items prop. If you have any questions on making this change, let me know.